draft-wiggers-hbs-state.md

```diff
@@ -390,7 +390,8 @@ main concerns here are
 A system may have a version of the private key stored in non-volatile memory
 (e.g. a disk) and will load it into volatile memory (e.g. RAM) while
 processing. Here, an implementer MUST ensure that these are always perfectly
-synchronized [MCGREW]. This can be particularly challenging if there are
+synchronized [MCGREW], meaning that no parts of the system are allowed to read
+any version of the key during procedure which load, write or modify keys. This can be particularly challenging if there are
```

Comment:
As a non-native speaker: du we need "procedures" or "procedure"

Reply:
This should be plural AFAIK

Reply:
as also pointed out by @crypto4a's comment

Reply:
Oh, now I see! Feels good to have, independently, the same thoughts as Jim.